src/Tribe/Cost_Utils.php

```diff
@@ -310,8 +310,15 @@ public function parse_cost_range( $costs, $max_decimals = null ) {
 		$costs = call_user_func_array( 'array_merge', $costs );
 
 		foreach ( $costs as $cost ) {
-			// Creates a Well Balanced Index that will perform good on a Key Sorting method
-			$index = str_replace( array( '.', ',' ), '', number_format( str_replace( $this->get_separators(), '.', $cost ), $max ) );
+			$numeric_cost = str_replace( $this->get_separators(), '.', $cost );
+
+			if ( is_numeric( $numeric_cost ) ) {
+				// Creates a Well Balanced Index that will perform good on a Key Sorting method
+				$index = str_replace( array( '.', ',' ), '', number_format( $numeric_cost, $max ) );
+			} else {
+				//
+				$index = sanitize_title( $numeric_cost );
```

Comment:
* Does this have potential to mean some indices will be 'well balanced' and some not? Does that have implications for use of min() and max() further on in the request?
* Comment is empty: add something or remove it!